README.md

```diff
@@ -1,217 +1,212 @@
-        // ...
-        
-        playAccountConfig = playAccountConfigs.defaultAccountConfig
-    }
+play {
+    jsonFile = file('your-key.json')
 }
 ```
 
 ## Configuration
 
-Once you have applied this plugin to your android application project you can configure it via the ```play``` block.
+Once you've setup the plugin, you can continue to configure it through the `play` block.
 
 ### Specify the track
 
-As a default your APK is published to the alpha track and you can promote it to beta or production manually. If you want to directly publish to another track you can specify it via the ```track``` property:
+By default, your app is plublished to the `alpha` channel where you can promote it to the beta or
+stable channels later from the Play Console. However, the Gradle Play Publisher plugin lets you
+choose another `track` if needed:
 
 ```groovy
 play {
     // ...
-    track = 'production' // or any of 'rollout', 'beta', 'alpha' or 'internal'
-    userFraction = 0.2 // only necessary for 'rollout', in this case default is 0.1 (10% of the target)
+    track = 'production' // Or any of 'rollout', 'beta', 'alpha' or 'internal'
+    userFraction = 0.2 // Only necessary for 'rollout'. The default is 0.1 (10%).
 }
 ```
 
-When defining the track as (staged) `rollout` you can also define a ```userFraction``` which is the portion of users who should get the staged rollout version of the APK.
+When initiating a staged release (`rollout`), the `userFraction` property will decide what
+percentage of your users should start recieving the update.
 
 ### Untrack conflicting versions
 
-The Google Play Developer API does not allow us to publish a beta version if there is an alpha version with a lower version code. If you want to publish to higher track and automatically disable conflicting APKs from a lower track, this can be specified by setting the `untrackOld` property to `true`.
+The Google Play Developer API does not allow a beta version to be published while there is an alpha
+version. If you want to automatically publish to a higher track and disable conflicting APKs from
+lower tracks, simply set the `untrackOld` property to `true`:
 
 ```groovy
 play {
     // ...
     track = 'beta'
-    untrackOld = true // will untrack 'alpha' while upload to 'beta'
+    untrackOld = true // Will untrack 'alpha' if needed to upload to 'beta'
 }
 ```
 
-This will untrack whatever versions are currently blocking the publishing process. That is: every APK with a version lower than the one being uploaded in any of the tracks lower than specified by `track`.
+`untrackOld` will untrack all versions blocking a publication: that is, every APK with a version
+code lower than the one being uploaded _and_ a lower track will be untracked. Example: publishing an
+APK with version code 42 to the beta channel will untrack versions 41 and lower from alpha and
+above. However, it will not untrack the stable channel or any alpha and above chanel with version
```

Comment:
"and above" seems wrong here. What is it referring to?

Reply:
I was referring to the `internal` track, but you're right, it's just confusing. Removing it.